src/global/global.h

```diff
@@ -54,7 +54,7 @@ typedef double Real;
 #define LOG_FILE_NAME "run_output.log"
 
 //Conserved Floor Values
-#define TEMP_FLOOR 1e-3 // in Kelvin
+#define TEMP_FLOOR 10 // in Kelvin
```

Comment:
should this change be wrapped in ifdef DUST so as to not change temp floor for others? not really sure, myself.